README.md

```diff
@@ -0,0 +1,158 @@
+
+### Tech Stack <a name="tech-stack"></a>
+
+  <ul>
+    <li>Ruby</li>
+  </ul>
+
+### Key Features <a name="key-features">
+
+-Decode Morse messages
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps:
+
+- Open your Terminal in your VSCode.
+- Navigate to the desired directory [folder] to which you want to clone the repository to
+
+### Prerequisites
+
+In order to run this project you need:
+
+- General Understanding Ruby syntax
+- basic Knowledge of Git Commands
+- basic Knowledge of the Terminal Commands
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- git clone https://github.com/MayPyone/Enumerable
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone https://github.com/MayPyone/Enumerable
+
+```
+
+--->
+
+### Install
+
+Must-Install-Project-with-the-Following-Programmes:
+
+- Git
+- Node.js
+- npm install
+  -Ruby
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Victor Correia**
+
+- GitHub: [@githubhandle](https://github.com/dvcorreia1984)
+- Twitter: [@twitterhandle](https://twitter.com/dvcorreia1984)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/dvcorreia)
+
+👤 **Victor Correia**
+
+- GitHub: [@githubhandle](https://github.com/dvcorreia1984)
+- Twitter: [@twitterhandle](https://twitter.com/dvcorreia1984)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/dvcorreia)
+
+👤 **May Pyone**
+
+- GitHub: [@githubhandle](https://github.com/MayPyone)
+- Twitter: [@twitterhandle](https://twitter.com/maypyone015)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/may-pyone-9439961a3/)
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Adding more functionality to the app.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please don't hesitate to support me with your valued contribution and don't forget to star the repo to reach more
+people.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank...
+
+- My Coding Partners
+- Microverse Community in Slack
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ❓ FAQ <a name="faq"></a>
+
+Can I clone this project?\*\*
+
+- Of course, you can as long as you fork it and star it too
+
+Can I contribute to this project?\*\*
+
+- definitely you can
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](LICENSE) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] The link to the MIT License is broken kindly fix it.

Reply:
@algerina Thank you. I added the licence file.